tests/yast2_gui/yast2_bootloader.pm

```diff
@@ -53,9 +55,11 @@ sub run() {
     type_string 'dummy-password';
     send_key 'alt-y';
     type_string 'dummy-password';
+    wait_still_screen 3;
 
     # OK => Exit
     send_key "alt-o";
+    wait_still_screen 600;
```

Comment:
creating initrd takes a long time and therefore it is on safe side to set 600 seconds. It won't take so long time, but if is still failed then we know that is an product issue anyway.